_posts/2023-08-25-using-kani-to-validate-security-boundaries-in-aws-firecracker.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: post
+title: Using Kani to Validate Security Boundaries in AWS Firecracker
+---
+
+Security assurance is paramount for any system running in the cloud. To take a step foward the highest levels of security, we have applied the [Kani model checker](https://github.com/model-checking/kani) to verify safety-critical properties in core components of the [Firecracker Virtual Machine Monitor](https://firecracker-microvm.github.io/) using mathematical logic.
```

Comment:
Just a suggestion:
```suggestion
Security assurance is paramount for any system running in the cloud. In order to achieve the highest levels of security, we have applied the [Kani model checker](https://github.com/model-checking/kani) to verify safety-critical properties in core components of the [Firecracker Virtual Machine Monitor](https://firecracker-microvm.github.io/) using mathematical logic.
```
(otherwise, replace "foward" with "toward")